packages/fxa-auth-server/lib/routes/auth-schemes/refresh-token.js

```diff
@@ -5,7 +5,7 @@
 'use strict';
 
 const AppError = require('../../error');
-const joi = require('@hapi/joi');
+const joi = require('joi');
```

Comment:
This feels a bit strange (but I see the "why").

We're using `require('joi')` now and setting to `joi`, `Joi`, or `isA` variable names.
Not sure if we want to settle on one (or even two, if refactoring from `isA`=>`Joi` isn't worth the effort; and it might not be) to be a bit more consistent.